adding notes for preprov challenges (#3959)
mesosphere/dcos-docs-site: pages/dkp/konvoy/2.1/choose-infrastructure/pre-provisioned/define-infrastructure/index.md

## Name your cluster

Give your cluster a unique name suitable for your environment.

Set the environment variable to be used throughout this documentation:

```sh
CLUSTER_NAME=my-preprovisioned-cluster
```

Note: If you want to create a unique cluster name, use this command.

> This will create a unique name every time you run it so use it with forethought.

Reviewer: ```suggestion
This creates a unique name every time you run it, so use caution when you run this command.
```

1.  Apply the new infrastructure file with the command:

    ```shell
    envsubst < preprovisioned_inventory.yaml | kubectl apply -f -
    ```

## Prepare your machines

> In order for DKP to fully install, it is recommended you stop `firewalld` if enabled so DKP can fully deploy.

Reviewer: ```suggestion
In order for DKP to fully install, we recommend you stop `firewalld`, if enabled, so that DKP can fully deploy.
```

> You will need to check to see if `firewalld` is running. SSH into all the machines you are deploying DKP onto, and check to see if it is running:

Reviewer: ```suggestion
Verify if `firewalld` is running by SSH into all machines you are deploying DKP onto, and check to see if it is running:
```

```sh
systemctl status firewalld
```

> If it is active, you will need to stop `firewalld`:

Reviewer: ```suggestion
If it is active, you need to stop `firewalld`:
```

```sh
systemctl stop firewalld
```

> 

Reviewer 2: We might also want to remove any preconfigured ip table filter rules as well

Author: @faiq good call - I'll iterate.
